trl/trainer/dpo_trainer.py

```diff
@@ -113,7 +113,7 @@ def __init__(
                 model = prepare_model_for_int8_training(model)
             model = get_peft_model(model, peft_config)
 
-        self.is_peft_model = getattr(model, "is_peft_model", False)
+        self.is_peft_model = getattr(model, "active_adapter", False)
```

Comment:
nit, this can be also:

```suggestion
        self.is_peft_model = is_peft_available() and isinstance(model, PeftModel)
```